tests/rptest/clients/rpk.py

```diff
@@ -62,10 +63,16 @@ def topic_line(line):
 
         assert False, "Unexpected output format"
 
+    def _get_brokers(self):
```

Comment:
the problem with this is that it doesn't support tests that dynamically change the cluster. I think you should drop this patch and instead add a lock to make the brokers method thread safe.

1. add a mutex to redpanda.py
2. add one line to redpanda::brokers `with lock.lock(): ...`
3. add a comment explaining that some tests use multiple thread on the coordinator node and should be re-written.

Reply:
Done!